types/tests/autogen.js

```diff
@@ -1,6 +1,10 @@
 import * as fs from 'fs';
 import * as path from 'path';
-import * as helpers from '../../src/helpers/index.js';
+import { fileURLToPath } from 'url';
+import * as helpers from '../../dist/helpers.esm.js';
+
+// eslint-disable-next-line es/no-import-meta
```

Comment:
why is this even a rule?

Reply:
It would be ok to disable the rule, it is for Internet Explorer.